app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -559,6 +567,10 @@ def rbac_user_delete_restriction?(user)
     ["admin", session[:userid]].include?(user.userid)
   end
 
+  def rbac_user_copy_restriction?(user)
+    user.userid == "admin"
```

Comment:
such method works for me on User object,
`
 User.first.super_admin_user?
`
and it looks like to me that variable user is kind_of(User)

will this work for you ?